data/fields/roof/shape.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "roof:shape",
+    "type": "combo",
+    "label": "Shape of building (part) roof",
```

```suggestion
    "label": "Roof shape",
```

Having those fields consistent "Roof …" is better IMO